dspace-api/src/main/java/org/dspace/app/util/DCInput.java

```diff
@@ -226,6 +227,15 @@ public DCInput(Map<String, String> fieldMap, Map<String, List<String>> listMap)
         relationshipType = fieldMap.get("relationship-type");
         searchConfiguration = fieldMap.get("search-configuration");
         filter = fieldMap.get("filter");
+        externalSources = new ArrayList<>();
+        String externalSourcesDef = fieldMap.get("externalsources");
+        if (StringUtils.isNotBlank(externalSourcesDef)) {
+            String[] sources = StringUtils.split(externalSourcesDef, ",");
```

Comment:
can we have any example, or use case where we use more than one external source? I'm asking this because I didn't see in any place the case of defining multiple external sources (it's comma separated), like:

`<externalsources>crossref, datacite</externalsources>`

I think it could be useful.

Reply:
There's an example for this in the test's config: https://github.com/DSpace/DSpace/pull/3109/files#diff-47993096c9f4173f69f5e9722712ea54753666c578f5b86b691021d973ce1e19R66
It's verified to work properly in this test: https://github.com/DSpace/DSpace/pull/3109/files#diff-6009af8825c771129ab9a3b5b7e3c104715cb2690ba72ff3cd814f2d496c6f6dR611